openapi/platform/platform.yaml

```diff
@@ -3976,6 +4011,38 @@ components:
                 title: Title of 987448ca field
                 description: Description for 987448ca field
                 fieldType: date
+              - fieldId: FieldID-987448f2-abc0-abc5-abc2-abc640bf4b00
+                dropdownValue: 'red'
+                title: Title of 987448f2 field
+                description: Description for 987448f2 field
+                fieldType: dropdown
+              - fieldId: FieldID-98744854-abc0-abc5-abc2-abc640bf4b01
+                currencyValue: 10000
+                title: Title of 98744854 field
+                description: Description for 98744854 field
+                fieldType: currency
+                currencyCode: CAD
+        Custom Fields with Dropdown Example:
+          data:
+            id: U-12345678-abcd-1234-abcd-123456789abc
+            type: userCustomFields
+            partnerFields:
+              - fieldId: FieldID-987448f2-abc0-abc5-abc2-abc640bf4b00
+                dropdownValue: 'red'
+                title: Title of 987448f2 field
+                description: Description for 987448f2 field
+                fieldType: dropdown
```

Comment:
How will developers using this API know what the valid values are? Will there be a field schema endpoint to list them?

Reply:
They will have to use the administration panel, I'm gonna add that to the description